.github/workflows/build.yml

```diff
@@ -3,7 +3,22 @@
 
 name: build
 
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+      - 'README_zh-CN.md'
+      - 'docs/**'
```

Comment:
We should not skip CI, when docs are modified. Because we will need readthedocs to rebuild the docs in the CI process.

Reply:
This only skips the process in build.yml.